js/mexc.js

```diff
@@ -405,6 +407,8 @@ module.exports = class mexc extends Exchange {
                 'info': currency,
                 'name': name,
                 'active': currencyActive,
+                'deposit': undefined,
```

Comment:
Please, add the same logic for currency-wide `deposit` and `withdraw` flags, derived from networks, as you did in the other exchanges.